src/USER-REAXC/reaxc_traj.h

```diff
@@ -42,7 +42,7 @@
 #define SCI_LINE  "%-37s%-24g\n"
 #define REAL3_LINE "%-32s%9.3f,%9.3f,%9.3f\n"
 
-#define INIT_DESC "%9d%3d%9s%10.3f\n" //AtomID - AtomType, AtomName, AtomMass
+#define INIT_DESC TAGINT_FORMAT "%3d%9s%10.3f\n" //AtomID - AtomType, AtomName, AtomMass
```

Comment:
good point. i'll do this differently. when looking closer, then this looks like it is trying to emulate some fortran style output, but the fixed buffer size will be missing blanks for atom IDs of 1 billion and overflow for atom IDs 10x larger. Similar for positions in very large boxes, it will be missing blanks for position values over 1000000 and under -100000, and overflow for 10x larger in magnitude.